htdocs/theme/eldy/style.css.php

```diff
@@ -2387,14 +2412,16 @@
 	border-left: 1px solid #CCC !important;
 	/* border-top: <?php echo 2; ?>px solid rgb(<?php echo $colortopbordertitle1; ?>) !important; */
 	border-top: <?php echo 2; ?>px solid rgb(<?php echo $colorbackhmenu1 ?>) !important;
+    	border-bottom: 1px solid #fff !important;
+	margin-bottom: -1px !important;
 }
 a.tab:hover
 {
 	/*
 	background: rgba(<?php echo $colorbacktabcard1; ?>, 0.5)  url(<?php echo dol_buildpath($path.'/theme/'.$theme.'/img/nav-overlay3.png',1); ?>) 50% 0 repeat-x;
 	color: #<?php echo $colortextbacktab; ?>;
 	*/
-	text-decoration: underline;
+	text-decoration: none;
```

Comment:
This is an accessibility error! Please keep in mind accessibility rules.
See https://www.sitepoint.com/15-rules-making-accessible-links/ (chapter 11) and https://www.w3.org/TR/WCAG20-TECHS/F73.html